style.css

```
@@ -0,0 +1,51 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-size: large;
```

I think we should use numeric value instead of using large keyword as it will affect the default in your website to make it responsive. 
```suggestion
  font-size: 12px;
```

---

- [ ] Your comment here is very nice and you told to the student the reason why he should do the request changes :+1: . Good job :smile: